src/mappings/factory.ts

```diff
@@ -48,6 +52,8 @@ export function handlePoolCreatedHelper(
     const bundle = new Bundle('1')
     bundle.ethPriceUSD = ZERO_BD
     bundle.save()
+
+    populateEmptyPools(event, poolMappings, whitelistTokens, tokenOverrides)
```

Comment:
nit - could we add some constant or something to only do this for OP 

like BACKFILL_SUPPORTED_CHAINS or something, not that urgent as we'll probably never have to backfill again